.travis.yml

```diff
@@ -6,18 +6,21 @@ cache:
   directories:
     - $HOME/.composer/cache
 
-php:
-  - '7.0'
+matrix:
+  include:
+    - php: '7.0'
+      env: PREFER_LOWEST=""
+    - php: '7.1'
+      env: PREFER_LOWEST="--prefer-lowest"
```

Comment:
you could use `COMPOSER_FLAGS` instead so you don't have to declare it for the first build instead of `PREFER_LOWEST=""`